private/optimize.m

```diff
@@ -97,6 +110,13 @@
 C = b/(1e-6*gamma^2*(problem.gMaxConstraint*1e-3/problem.dt)^2*(problem.totalTimeActual*1e-3)^3);
 kappa = 4*C;
 
+if 0
+    plot(g)
+    title(['Final time = ' num2str(optimizationTime, 2)])
+    drawnow
+    pause(.2)
+end
```

Comment:
Remove

Reply:
Yes, please do. I just used this to understand the optimizer during some weird behaviour.